--!strict
--[[
    UI Screens
    Full screen UI views
]]

local Screens = {}

Screens.FarmHUD = require(script.FarmHUD)
Screens.SeedPanel = require(script.SeedPanel)
Screens.PlotInfoPanel = require(script.PlotInfoPanel)
Screens.NotificationSystem = require(script.NotificationSystem)
Screens.PetInventory = require(script.PetInventory)
Screens.PetDetails = require(script.PetDetails)
Screens.PetPenUI = require(script.PetPenUI)
Screens.PetSelectModal = require(script.PetSelectModal)

return Screens
